pkg/pulumiyaml/analyser_test.go

```diff
@@ -284,7 +284,6 @@ func TestTypePropertyAccess(t *testing.T) {
 			expectedType: "Invalid",
 			errMsg: `Cannot access into start of type Union<List<string>, Map<number>, {foo: List<any>}>:'start' could be a type that does not support accessing:
   Array<string>: cannot access a property on 'start' (type List<string>)
-  Map<number>: cannot access a property on 'start' (type Map<number>)
```

Comment:
This error says we can't index into a map. However, we now can, so the error we would encounter (that "start" is not a number) will now come later.